pandas/core/window.py

```diff
@@ -1005,6 +1017,11 @@ def sum(self, *args, **kwargs):
 
     _shared_docs['max'] = dedent("""
     Calculate the %(name)s maximum.
+
+    Parameters
+    ----------
+    *args, **kwargs :
```

Comment:
can you remove all the colons at the end of args and kwargs (in all docstrings of this PR)? when there is no type, there shouldn't be colons.

Reply:
will do, thanks